src/Dapr.AzureFunctions.Extension/Bindings/DaprPubSubEvent.cs

```diff
@@ -17,13 +17,25 @@ public class DaprPubSubEvent
         /// Initializes a new instance of the <see cref="DaprPubSubEvent"/> class.
         /// </summary>
         /// <param name="payload">The payload of the outbound pub/sub event.</param>
+        /// <param name="name">The pub/sub name of the outbound pub/sub event.</param>
         /// <param name="topic">The topic of the outbound pub/sub event.</param>
-        public DaprPubSubEvent(JToken payload, string? topic = null)
+        public DaprPubSubEvent(JToken payload, string? name = null, string? topic = null)
```

Comment:
note: this is a pretty easy to miss breaking change for callers. Anyone who's written `new DaprPubSubEvent(..., "foo")` will need to recognize and react. It's the right API design to have name before topic though...

Reply:
Do we want the name and topic to be nullable and have default values if we expect them to be non-null?

Reply:
They are allowed to be null here - there's a way ways to use this type.

You could specify a name and topic here.... or you could rely on them to specified with attributes. Example here: https://github.com/dapr/azure-functions-extension/pull/57/files#diff-f880168c1bd7a29acd83c4fcb93b9e47R145